docs/content/get-started/kubernetes-platform/supported-platforms.md

```diff
@@ -9,9 +9,9 @@ docs: "DOCS-688"
 
 ## Supported Platforms
 
-NGINX Service Mesh supports Kubernetes versions 1.18-1.22.
+NGINX Service Mesh supports Kubernetes versions 1.24-1.26.
```

Comment:
Probably be easiest to just link to the supported versions table.

Reply:
"Support" is a powerful word.  It will result in customers not trying the product on versions <1.24
I don't think the intention from the internal conversations is to use the term "supported" to indicate only those versions we have tested against.  Even when we are confident there is nothing breaking in versions back to 1.18 (there have been no related API or feature changes that we are aware of that would break behavior of the Mesh).

We should not be reflecting our test matrix as a supported statement, but rather where a K8s release introduces some behavior change that we absolutely have to patch for.  That should drive the supported versions.

In this case, we have a dependency on a version of NIC, therefore we could pull in and align with the NIC K8s version support  1.12 - current.

Reply:
@sjberman, I removed the k8s and openshift versions from the tech specs table since we are already documenting it here on the supported platform page. It eliminates maintaining the version in two separate places or having users jump to a different page to see it. Having the k8s version here on the supported platform page made more sense to me.

@brianehlert, updated to use the same phrasing as the OpenShift page: saying `tested` instead of `supported`.  On the next line down there is a clarification that it may work on other versions.

Reply:
The tech specs doc should be our source of truth, since we link to it in multiple places. We reduced supported version mentions in the past to consolidate them in the table, but we missed the couple instances that you updated.

Reply:
Updated to link to the tech specs. It's not a good user experience to have to jump around through the docs so much, but I suppose maintainability counts too.

Reply:
The word problem is "supports" - the OpenShift phrasing is better 
until we can get better knowledge and alignment on breaking dependent changes.

Reply:
This section has been updated so this comment thread is out of date. Please see the latest changes.